src/app/api/services/dot-push-publish-dialog/dot-push-publish-dialog.service.ts

```diff
@@ -1,16 +1,18 @@
 import { Injectable } from '@angular/core';
 import { Observable, Subject } from 'rxjs';
-import { DotPushPublishEvent } from '@models/push-publish-data/push-publish-data';
+import { DotPushPublishDialogData } from '@models/dot-push-publish-dialog-data/dot-push-publish-dialog-data.model';
 
 @Injectable()
 export class DotPushPublishDialogService {
-    private _showDialog: Subject<DotPushPublishEvent> = new Subject<DotPushPublishEvent>();
+    private _showDialog: Subject<DotPushPublishDialogData> = new Subject<
```

Comment:
why "_"?

Reply:
used to be like that, to differentiate between that and `showDialog$`  which is in the same class.

Reply:
changed.